nvfuser/csrc/compute_at_map.cpp

```diff
@@ -431,7 +431,7 @@ void IterDomainGraph::build(Fusion* fusion) {
             // might not be a compute at leaf domain of `p_tv`, but it actually
             // has an equivalent compute at leaf domain. For that case, we map
             // the equivalent compute at leaf domain.
-            for (int i = 0; i < p_tv->getComputeAtPosition(); i++) {
+            for (int i = 0; i < int(p_tv->getComputeAtPosition()); i++) {
```

Comment:
nit: Why do we need to create new int here is the return from this method, unsigned int ? Maybe its better to use i as unsigned ?